src/pages/ContractInfoPage/components/ContractInfo/ContractInfoHeader.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div class="contract-info-header">
```

Comment:
if it is header use semantically correct tag `<header>`

Reply:
done